bin/node-template/pallets/template/src/lib.rs

```diff
@@ -75,6 +75,7 @@ decl_module! {
 		/// Just a dummy entry point.
 		/// function that can be called by the external world as an extrinsics call
 		/// takes a parameter of the type `AccountId`, stores it, and emits an event
+		#[weight = frame_support::weights::SimpleDispatchInfo::default()]
```

Comment:
Are we sure we want notion of `default` at all? It is not much different putting this default instead of forgetting to put something at all.

Reply:
I don't think it is critical to act upon that for now as it is semantically the same and in the hopefully very soon feature, we won't have any of them in the code base.